### ISSUE TITLE
Following clonotypes between samples 

### ISSUE THREAD

**Issue by ASNbioinf** [ncborcherding/scRepertoire#428]
I have a similar problem to #40. I have T cells that I clustered. There are clones shared between samples that belong to the same patient. I used `clonalCompare()` to show the **proportion** of each clone within the specified samples. Is there a function that allows me to plot the **frequency** of each clone within the specified samples instead of **proportion**? I mean, I like this plot, but I want to put the **frequency** in the y-axis.

**Comment by Qile0317** [comment]
@ncborcherding I don't believe there is an option to do this yet but the change should be pretty trivial - I can whip up a PR for this.

**Comment by ncborcherding** [comment]
@Qile0317, 

That would be awesome if you can include it in the work you are doing. I think having an option to plot frequency or proportion in ```clonalCompare()``` would be perfect and then users could use **exportTable** for either.

Nick

**Comment by Qile0317** [comment]
@ncborcherding Just to make sure - are the proportions strictly supposed to be (Frequency of all clones) / (number of clones)? Trying to figure out if my code is incorrect or (probably not) the entire function has been doing faulty computations this whole time.

**Comment by ASNbioinf** [comment]
Proportion = count of clone / total clones in sample

**Comment by ncborcherding** [comment]
@Qile0317 Proportion = frequency of individual clone / total number of sequences in a sample.

**Comment by ncborcherding** [comment]
Fix is up in the dev version, we will do some more testing and it will be in the main branch soon. I'm going to close this issue.
